src/components/KBadge/KBadge.vue

```diff
@@ -50,26 +50,11 @@ import { ref, computed, watch, useAttrs, PropType } from 'vue'
 import KButton from '@/components/KButton/KButton.vue'
 import KIcon from '@/components/KIcon/KIcon.vue'
 import KTooltip from '@/components/KTooltip/KTooltip.vue'
-import type { BadgeAppearance, BadgeAppearanceRecord, BadgeShapeRecord, BadgeShape } from '@/types'
+import { BadgeAppearance, BadgeShape, appearances, shapes } from '@/types'
 import useUtilities from '@/composables/useUtilities'
 
 const { getSizeFromString } = useUtilities()
 
-export const appearances: BadgeAppearanceRecord = {
-  default: 'default',
-  success: 'success',
-  danger: 'danger',
-  info: 'info',
-  warning: 'warning',
-  custom: 'custom',
-  neutral: 'neutral',
-}
-
-export const shapes: BadgeShapeRecord = {
-  rounded: 'rounded',
-  rectangular: 'rectangular',
-}
```

Comment:
question: These exports were moved to the `badge.ts` file; however, I don't see them being used in this file anymore...? 

Also, why are you making type interface changes on a PR for tokens? This should likely be reverted as the changes are unrelated.